### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by ghost**
Fixing some typos and grammatical mistakes in the comments in the class `TelegramLogHandler`

### CODE REVIEW

File: Sources/LoggingTelegram/TelegramLogHandler.swift [hunk 1]
```diff
@@ -60,10 +60,11 @@ public class TelegramLogHandler<T>: LogHandler where T: TelegramId {
 
   /// Creates a `TelegramLogHandler` for sending `Logger` output directly to Telegram.
   /// - Parameters:
-  ///   - label: The log label for he log handler.
-  ///   - chats: Target chats to send.
+  ///   - label: The log label for the log handler.
+  ///   - chat: Target chats to send to.
   ///   - token: Telegram Bot Token.
   ///   - level: (Optional) The minimal log level for this logger.
+  ///   - mute: (Optional)  Sending this message to the user silently
```
Comment (stevapple) [inline Sources/LoggingTelegram/TelegramLogHandler.swift]: The additional whitespace may be unnecessary?

File: Sources/LoggingTelegram/TelegramLogHandler.swift [hunk 2]
```diff
@@ -60,10 +60,11 @@ public class TelegramLogHandler<T>: LogHandler where T: TelegramId {
 
   /// Creates a `TelegramLogHandler` for sending `Logger` output directly to Telegram.
   /// - Parameters:
-  ///   - label: The log label for he log handler.
-  ///   - chats: Target chats to send.
+  ///   - label: The log label for the log handler.
+  ///   - chat: Target chats to send to.
```
Comment (stevapple) [inline Sources/LoggingTelegram/TelegramLogHandler.swift]: `chats` is also a typo here, you may fix it at the same time.

**Review by stevapple — CHANGES_REQUESTED** [review]

File: Sources/LoggingTelegram/TelegramLogHandler.swift (hunk 1 above)
Comment (ghost) [reply Sources/LoggingTelegram/TelegramLogHandler.swift]: Sorry I made a mistake here... The second space is unnecessary. I'll fix it

File: Sources/LoggingTelegram/TelegramLogHandler.swift (hunk 2 above)
Comment (ghost) [reply Sources/LoggingTelegram/TelegramLogHandler.swift]: Oh I'll fix it.

> `chats` is also a typo here, you may fix it at the same time.

File: Sources/LoggingTelegram/TelegramLogHandler.swift [hunk 3]
```diff
@@ -60,10 +60,11 @@ public class TelegramLogHandler<T>: LogHandler where T: TelegramId {
 
   /// Creates a `TelegramLogHandler` for sending `Logger` output directly to Telegram.
   /// - Parameters:
-  ///   - label: The log label for he log handler.
-  ///   - chats: Target chats to send.
+  ///   - label: The log label for the log handler.
+  ///   - chat: The target chat to send to.
   ///   - token: Telegram Bot Token.
   ///   - level: (Optional) The minimal log level for this logger.
+  ///   - mute: (Optional) Sending this message to the user silently
```
Comment (stevapple) [inline Sources/LoggingTelegram/TelegramLogHandler.swift]: Here's a missing dot.

**Review by stevapple — CHANGES_REQUESTED** [review]

File: Sources/LoggingTelegram/TelegramLogHandler.swift (hunk 3 above)
Comment (ghost) [reply Sources/LoggingTelegram/TelegramLogHandler.swift]: Already fixed with a new commit.

**Review by stevapple — APPROVED** [review]

**Pull request merged**